webview/src/plots/components/comparisonTable/styles.module.scss

```diff
@@ -189,22 +189,48 @@ $gap: 4px;
   display: none;
 }
 
+.classButtons {
+  font-family: $font;
+  display: flex;
+  align-items: center;
+  gap: 6px;
+}
+
+.tableRowClassButtons {
+  max-width: calc(100% - 180px);
```

Comment:
What does the `180px` come from? Can you create a variable for it so that its name give some context?

Reply:
It's the combined width of the "Classes" title and "Show more" button. Good idea, I'll create a variable.